### PR TITLE
Navbar cleanup

### PR DESCRIPTION
**Pull request by technicalex**
This pull request moves styling for the new header and footer into the main.css file. The results of these changes have been compiled into main-giraffe_navbar_cleanup.css on S3. Additional pull requests will be opened in the ak-template-set and front-wordpress repositories to remove styling from the header and footer files. I'll add additional comments inline.

### CODE REVIEW

File: src/styles/components/_nav.scss [hunk 1]
```diff
@@ -385,4 +385,418 @@
     color: $azure;
     font-weight: 600;
   }
+}
+
+#main-header-2022 {
```
Comment (technicalex) [inline src/styles/components/_nav.scss:390]: Note that old header and footer styles are still included in these files, so the pages which have not yet applied the new header and footer (via main-giraffe) will not look completely broken.

**Review by technicalex — COMMENTED** [review]
Annotations!

File: src/styles/components/_header.scss [hunk 2]
```diff
@@ -54,9 +54,15 @@
 
 // You can use either a '.giraffe>body' or a .giraffe#wrapper to add padding for the nav
 body, &#giraffe-wrapper {
```
[the remark this answers is not in the document]
Comment (strangeways) [reply src/styles/components/_header.scss:56]: I agree that the new header should be the default and the old header can be the exception.

**Review by strangeways — APPROVED** [review]
👍

**Comment by technicalex** [comment]
Fixed issue flagged by Sonarcloud! Really cool to see it surface actual issues.

The recomputed main-giraffe_navbar_cleanup.css file is on S3 and also used on [this test page](https://act.moveon.org/survey/alex-test-survey-2022/).

**Pull request merged**